pandas/core/generic.py

```diff
@@ -7381,6 +7381,7 @@ def asfreq(
         Parameters
         ----------
         freq : DateOffset or str
+            frequency string
```

Comment:
Not only a string, can also be a DateOffset object

(also, can you have it start with a capital + end with a period?)